### DISCUSSION TITLE
Welcome to Seurat Discussions!

### DISCUSSION THREAD

**Discussion by satijalab** [satijalab/seurat#3995] [General]
This is the place to ask your questions related to the analysis of single-cell data or the use of the Seurat package. Anyone in the community is welcome to contribute to the discussions here. If you see a question that you know the answer to, feel free to post your answer!

For bug reports or requests for new features, please open a github issue instead.

**Comment by ImtiazKhuda** [comment]
Hello, I am running "Interoperability between single-cell object formats". I want to convert seurat object to SingleCellExperiment object but always getting the same error.
_pbmc.sce <- as.SingleCellExperiment(pbmc)
Error in UseMethod(generic = "as.SingleCellExperiment", object = x) :  no applicable method for 'as.SingleCellExperiment' applied to an object of class "seurat"._

But when converting SingleCellExperiment object to seurat object that is working perfectly. Otherwise, everything working perfectly. Does anyone have any idea, what is wrong? Thank you!

**Reply by timoast** [reply]
Please open a new issue
